tests/test_get_parser_unit.py

```diff
@@ -156,6 +156,8 @@ def all_test_args():
         "--use-pnfs-dropbox",
         "--verbose",
         "--devserver",
+        "--onsite-only",
+        "--offsite",
```

Comment:
Agh, I forgot one last thing.  My mistake.  Down in test_get_parser_all, please add _either_ one of `"--onsite-only"` or `"--offsite"` to the `args_exclude_list` in the `test_get_parser_all` function.  

When these unit tests were first written, there were no mutually exclusive argument groups.  When I added the first one for singularity stuff, I had to add that list (and exclude any arg in that list from the test) to make the unit test work.

Thanks!

Reply:
The requested change should be made on line 225, as I see it here.